tsconfig.packages.json

```diff
@@ -1,15 +1,22 @@
 {
   "compilerOptions": {
     "composite": true,
-    "declaration": true,
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
+    "lib": ["DOM"],
     "module": "CommonJS",
     "moduleResolution": "node",
-    "useUnknownInCatchVariables": false,
-    "sourceMap": true,
+    "paths": {
+      "@metamask/*/test-utils": ["../*/src/test-utils"],
+      "@metamask/*": ["../*/src"]
+    },
+    "resolveJsonModule": true,
     "strict": true,
-    "target": "ES2017"
-  },
-  "exclude": ["**/__snapshots__/**", "**/test/**", "**/*.test.ts"]
+    "target": "ES2017",
+    "useUnknownInCatchVariables": false,
+
+    // TODO: Move this to `tsconfig.build.json`.
```

Comment:
Should we do this now?

Reply:
I already did, but forgot to remove it from here. Will do that now.